<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserForm</class>
 <widget class="QDialog" name="UserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Usuario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDni">
       <property name="text">
        <string>DNI:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditDni"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelNombre">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditNombre"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelTelefono">
       <property name="text">
        <string>Teléfono:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditTelefono"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelEmail">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditEmail"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelDepartamento">
       <property name="text">
        <string>Departamento:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBoxDepartamento"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelEmpresa">
       <property name="text">
        <string>Empresa:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBoxEmpresa"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelEstado">
       <property name="text">
        <string>Estado:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBoxEstado"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelRol">
       <property name="text">
        <string>Rol:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="comboBoxRol"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelFoto">
       <property name="text">
        <string>Foto (ruta):</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEditFoto"/>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="btnSeleccionarFoto">
       <property name="text">
        <string>Seleccionar Foto</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="labelFotoRuta">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelPreviewImage">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>150</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>Previsualización</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnGuardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
